### PR TITLE
[glossary] adding draft of concept: champion

### PR DESCRIPTION
**Pull request by vsoch**
This will add "champion" to be one of the concepts to be under review!

Signed-off-by: vsoch <vsochat@stanford.edu>

### CODE REVIEW

File: community/concepts/champion.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Champion
+
+A champion is an individual that starts an initiative with a determined promise
+to finish it. The individual generally takes charge of a well-scoped task, and
+provides clear direction and confidence about completing it. By calling someone
+a champion, you are placing faith in their ability as a leader and visionary.
+You can read more about champions [here](concepts/champion.md)
```
Comment (s-ben) [inline community/concepts/champion.md]: This link is 404'ing for me, but presumably that's just because this was broken off from the bigger initial PR?

Comment (vsoch) [reply community/concepts/champion.md]: I think originally this glossary file was one level up, and would link to details in the file referenced (which is now the one we are commenting on) so it should be removed.

Comment (vsoch) [reply community/concepts/champion.md]: Fixed.

**Comment by vsoch** [comment]
Great!

**Pull request merged**